common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -36,13 +36,18 @@
 
 @ConfigCategory(Category.UI)
 public class WynncraftButtonFeature extends Feature {
-    private static final String GAME_SERVER = "play.wynncraft.com";
-    private static final String LOBBY_SERVER = "lobby.wynncraft.com";
+    private static final String GAME_SERVER = "play";
+    private static final String BETA_SERVER = "beta";
+    private static final String LOBBY_SERVER = "lobby";
```

Comment:
Technically we also have "hub"

Reply:
Is that different from lobby?

Reply:
No, just a secondary name.

Reply:
(For connecting it should not matter, I forgot this is not a model..)